.github/actions/prepare-low-bandwidth/action.yml

```diff
@@ -0,0 +1,106 @@
+name: Prepare Low Bandwidth Environment (MacOS & iOS Simulators only)
+description: prepare any workflow for low bandwidth testing
+
+inputs:
+  test_server_url:
+    description: The URL of the test server
+    required: true
+  device_name:
+    description: The iOS simulator name
+    required: true
+  download_speed:
+    description: The download speed limit (in Kbit/s)
+    required: false
+    default: "3300"
+  upload_speed:
+    description: The upload speed limit (in Kbit/s)
+    required: false
+    default: "3300"
+  latency:
+    description: The latency (in ms) each way
+    required: false
+    default: "500"
+
+
+runs:
+  using: composite
+  steps:
+      - name: delete the zip file and trash (to free up space)
+        shell: bash
+        run: |
+          rm -rf mobile-artifacts/*.zip
+          sudo rm -rf ~/.Trash/*
```

Comment:
Echoing what I mentioned for the `dnctl` failure: the [no space left on device](https://github.com/mattermost/mattermost-mobile/actions/runs/11011189890/job/30576168778#step:7:225) error doesn't seem to be related to disks (which are at ~0% usage), perhaps if the root cause is found, these these steps related to disk space may be removed

Reply:
I agree, I've seen folks with no space left on device on github action, but they tend to be something to do with docker.  I'm not using docker container, so I'm still at a lost right now.